mmf/utils/visualize.py

```diff
@@ -45,3 +48,27 @@ def visualize_images(
 
     plt.axis("off")
     plt.imshow(grid.permute(1, 2, 0))
+
+
+def visualize_frcnn_features(image_path, features_path, objids, attrids):
+    img = np.array(Image.open(image_path))
```

Comment:
Isn't this technically same as `img_tensorize`?

Reply:
Actually yes, I am realizing I can actually just remove that line and pass in the image path directly